classwork/03-classwork.qmd

```diff
@@ -101,16 +101,16 @@ This is how you'd fit a decision model:
 ```{r tree_wflow}
 tree_spec <-
   decision_tree() %>% 
-  set_mode("regression")
+  set_mode("classification")
 
 tree_wflow <- workflow() %>%
-  add_formula(latency ~ .) %>%
+  add_formula(tip ~ .) %>%
   add_model(tree_spec)
 
 tree_wflow
 ```
 
-Now use a similar approach to fit a linear model! Call it `lm_wflow`.
+Now use a similar approach to fit a different model!
```

Comment:
We could be prescriptive about the model here and ask them for a logistic regression.

Reply:
I would be game for that

Reply:
edited in [c694e26](https://github.com/tidymodels/workshops/pull/109/commits/c694e261bc079940b36cc85f849d8df6ce60c367) ✔️